packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2608,7 +2608,7 @@ export interface NetworkDiagnosticsState {
 }
 
 // @beta
-export type OnFetchProfileCallback = (userId: string) => Promise<Profile | undefined>;
+export type OnFetchProfileCallback = (userId: string, displayName?: string) => Promise<Profile | undefined>;
```

Comment:
to make this extendable, should this be profileData instead of displayName?
```suggestion
export type OnFetchProfileCallback = (userId: string, profileData?: Profile) => Promise<Profile | undefined>;
```